stock_picking_mass_action/wizard/mass_action.py

```diff
@@ -56,7 +56,7 @@ def _default_picking_ids(self):
     picking_ids = fields.Many2many(
         string='Pickings',
         comodel_name="stock.picking",
-        default=lambda self: self._default_picking_ids(),
+        default=_default_picking_ids,
```

Comment:
@carlosdauden Why changing lambda to non inheritable method ?

Reply:
I thought I had read that it was not necessary anymore, but surely I am wrong because I just tried it and still need.

Changes done.